src/plugin.js

```diff
@@ -1,42 +1,72 @@
 import { createStore } from './store'
 
-// Vanilla Installer
+export const injectedPropertyName = 'stator'
 
-function lazyInject (Vue, key, setter) {
-  // Check if plugin not already installed
-  const installKey = `__stator_${key}_installed`
-  if (Vue[installKey]) {
-    return
-  }
+export function injectStator (Vue, options) {
+  lazyInject(Vue, injectedPropertyName, (vm) => {
+    return createStore(options, vm.$root)
+  })
 
-  Vue[installKey] = true
+  injectProperty(Vue, injectedPropertyName, '$state')
+  injectProperty(Vue, injectedPropertyName, '$getters')
+  injectProperty(Vue, injectedPropertyName, '$actions')
+}
 
-  // Call Vue.use() to install the plugin into vm
+export function lazyInject (Vue, key, setter) {
   Vue.use(() => {
```

Comment:
I have removed the installed key's check here. Although these methods are exported, those exports are only used in the tests. We dont expect them to the user. Which means the only entry point for the user is the default export which also is a Vue-plugin and I've added the install keys there.